lib/inventory/inventory_api.rb

```diff
@@ -56,15 +54,13 @@ def list_resource_types(feed = nil)
         the_feed = hawk_escape feed
         ret = http_get('/feeds/' + the_feed + '/resourceTypes')
       end
-      val = []
-      ret.each { |rt| val.push(ResourceType.new(rt)) }
-      val
+      ret.map { |rt| ResourceType.new(rt) }
     end
 
     # List the resources for the passed feed and resource type. The representation for
     # resources under a feed are sparse and additional data must be retrived separately.
     # It is possible though to also obtain runtime properties by setting #fetch_properties to true.
-    # @param [String] feed The id of the feed the type lives under. Can be nil for feedless types
+    # @param [String] feed The id of the feed the type lives under. Can be nil for all feeds
```

Comment:
I was using the word feedless to indicate that things like the URLs in Hawkular do not have a feed as opposed to e.g. WildFlys to have a feed. My wording was probably not good - we need something better.